packages/animations/test/modal_test.dart

```diff
@@ -321,23 +321,20 @@ class _FlutterLogoModalState extends State<_FlutterLogoModal> {
 
 class _TestModalConfiguration extends ModalConfiguration {
   _TestModalConfiguration({
+    Color barrierColor = Colors.green,
```

Comment:
same here.

Reply:
We can make these non-configurable, since it's just a test configuration